### DISCUSSION TITLE
Backup, transfer notes from one TB to another if TB crashed

### DISCUSSION THREAD

**Discussion by yardstudio** [xnotepp/xnote#107] [Q&A]
Hello, 

I am using portable TB many years and as versions going to up, then some features and addon´s top working, some new arrived.. life cycle. Unfortunately, sometime TB crashing after many updates and fix is hard. 
Then newly  fresh installation is solution. But, if I am using XNote++ on previous profile, is any way to backup it and transfer to another profile (to the same email account)? How are notes linked with mails?  This is some uniquie mail ID, which is different in different profiles (the same mail message)

Or simply need starting noted again and again :) and lost my old notes?

Thanks for great plugin!

Regards 
Jaro

**Comment by opto** [comment]
the mail id is the same in all profiles. So you just need to copy the XNOTe folder to the new profile.
